### PR TITLE
F/cloud 1900 pre commit

### PR DESCRIPTION
**Pull request by naveen-variant**
# Description

Changing to pre-commit.

Fixes [#22](https://usxtech.atlassian.net/browse/CLOUD-1900)

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## How Has This Been Tested?

[Branch Link](https://github.com/variant-inc/demo-python-flask-variant-api/tree/test/master-postgress) 
[Sonarcloud Link](https://sonarcloud.io/summary/new_code?id=variant-inc_demo-python-flask-variant-api&branch=test%2Fmaster-postgress) 
[Octopus Link](https://octopus.apps.ops-drivevariant.com/app#/Spaces-2/projects/demo-test-postgress/deployments/releases/0.1.0-test-master-post0001.417/deployments/Deployments-43207?activeTab=taskLog) 

- [ ] Sanity Testing

## Checklist

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added documentation to test

### CODE REVIEW

File: .pre-commit-config.yaml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+repos:
```
Comment (vibindaniel) [inline .pre-commit-config.yaml:1]: How is this different from running super linter in actions?

Comment (naveen-variant) [reply .pre-commit-config.yaml:1]: Removed super linter.

**Comment by vibindaniel** [comment]
Reduce number of commits. I don't think 6 are required.

**Comment by naveen-variant** [comment]
> Reduce number of commits. I don't think 6 are required.

Done

**Review by vibindaniel — APPROVED** [review]

**Pull request merged**